src/helpers/git_helpers/remote_helpers.sh

```diff
@@ -23,7 +23,7 @@ function remote_domain {
 # Returns the USER/REPO for the remote repository
 function remote_repository_name {
   local domain=$(remote_domain)
-  remote_url | sed -E "s#.*$domain[/:](.+)#\1#" | sed "s/\.git$//"
+  remote_url | sed -E "s#.*$domain[/:](.+)#\1#" | sed "s/\.git$//" | sed 's#ssh://git@[^/]*/##'
```

Comment:
Thanks @charlierudolph,  bug I think that `$domain` is "ssh://github.com" not "github.com"
Should I modify the `remote_domain` function ?

```bash
echo "ssh://git@github.com/Originate/git-town" |sed -E "s#(https?://([^@]*@)?|git@)([^/:]+).*#\3#"
# outputs ssh://github.com
```